modules/nw-install-config-gateway-api.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-config-gateway-api_{context}"]
+= Installing and configuring Contour for Gateway API
+
+The following guide provides instructions for using link:https://gateway-api.sigs.k8s.io/[Gateway API] with the Contour on {product-title}.
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `oc` command line.
+* You installed the Contour Operator.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install Contour configured for Gateway API and dependent resources:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.16.0/examples/gateway/gateway.yaml
+----
++
+[NOTE]
+====
+Envoy is exposed using a LoadBalancer service. Replace `gateway.yaml` with `gateway-nodeport.yaml` to use a NodePort service instead.
+====
+
+. Verify that all pods in the `projectcontour` namespace are running:
++
+[source,terminal]
+----
+$ oc get po -n projectcontour
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                         READY   STATUS      RESTARTS   AGE
+contour-768547cfb8-c2rhn     1/1     Running     0          2m
+contour-768547cfb8-q866f     1/1     Running     0          2m
+contour-certgen-main-rb2h2   0/1     Completed   0          92s
+envoy-d5djm                  2/2     Running     0          2m41s
+envoy-gjwz5                  2/2     Running     0          2m41s
+envoy-hbg6j                  2/2     Running     0          2m41s
+----
++
+The number of Envoy pods depends on how many worker nodes are in your cluster.
++
+[NOTE]
+====
+It can take a few minutes for the Contour Operator to become available.
+====
```

Comment:
This belongs in nw-install-contour-operator.adoc.

Reply:
Thank you! I'll get that updated.